app/src/main/java/com/example/coinage/fragments/TransactionListFragment.java

```diff
@@ -63,14 +63,18 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         button2.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                FragmentTransaction fragmentTransaction = getActivity()
-                        .getSupportFragmentManager().beginTransaction();
-                fragmentTransaction.replace(R.id.frameLayout, new SpendingLimitOverviewFragment());
-                fragmentTransaction.commit();
+                goOverview();
             }
         });
     }
 
+    private void goOverview() {
+        FragmentTransaction fragmentTransaction = getActivity()
+                .getSupportFragmentManager().beginTransaction();
+        fragmentTransaction.replace(R.id.frameLayout, new SpendingLimitOverviewFragment());
```

Comment:
Love the descriptive name SpendingLimitOverviewFragment!

Reply:
Oh oops I should probably update the function name too